cmd/piper.go

```diff
@@ -199,16 +199,21 @@ func PrepareConfig(cmd *cobra.Command, metadata *config.StepData, stepName strin
 
 	config.MarkFlagsWithValue(cmd, stepConfig)
 
-	for name, v := range stepConfig.HookConfig {
-		if name == "sentry" {
-			hookConfig, _ := v.MarshalJSON()
-			_ = json.Unmarshal(hookConfig, &GeneralConfig.HookConfig.SentryConfig)
-		}
-	}
+	retrieveHookConfig(stepConfig.HookConfig, &GeneralConfig.HookConfig)
 
 	return nil
 }
 
+func retrieveHookConfig(source *json.RawMessage, target *HookConfiguration) {
+	if source != nil {
+		log.Entry().Info("Retrieving hook configuration")
+		err := json.Unmarshal(*source, target)
+		if err != nil {
+			log.Entry().Warningf("Failed to retrieve hook configuration: %v", err)
```

Comment:
I'm not familiar with the details here, but is it OK to continue after having a failure wrt hook configuration?

Would't is be easier in terms of troubleshooting efforts to stop here rather than to continue and to fail later on with some other error which is basically triggered here?

Reply:
the hook is about error logging.
It will not break in case s.th. is wrong here.

Reply:
so far an Unmarshalling error was just ignored `_ = ...`, now we at least write a warning ...